unliftio-core/src/Control/Monad/IO/Unlift.hs

```diff
@@ -160,3 +161,15 @@ wrappedWithRunInIO :: MonadUnliftIO n
                    -> m b
 wrappedWithRunInIO wrap unwrap inner = wrap $ withRunInIO $ \run ->
   inner $ run . unwrap
+
+{- | A helper function for lifting @IO a -> IO b@ functions into any @MonadUnliftIO@.
+
+=== __Example__
+
+> liftedTry :: (Exception e, MonadUnliftIO m) => m a -> m (Either e a)
+> liftedTry m = liftIOOp Control.Exception.try m
+-}
```

Comment:
Can you add a minor version bump, since comment, and changelog entry?

Reply:
Or rather: update changelog with the new minor version

Reply:
Done :ok_hand: